### ISSUE TITLE
Sharding fixes the former Trash Cube pAI bug... somehow

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#19533]
(WEB REPORT BY: mask2697 REMOTE: 172.93.109.202:7777)
# Revision
4a8bd3e7ae934b0fab2c1454c643ae426a9f8b0d

# Description

Getting sharded while holding a trash cube pAI fixes the trash cube pAI rather than killing it
# Steps to Reproduce

Mentioned above
# What you Expected

Death
# What Actually Happened

Not death??

**Comment by Exxion** [comment]
Like it just drops on the floor?

**Comment by SGuivenne** [comment]
yes, the pai just drops on the floor as a now regular PAI again.
